misc/openapi-ballerina/modules/openapi-validator/src/main/java/org/ballerinalang/openapi/validator/MatchResourcewithOperationId.java

```diff
@@ -0,0 +1,534 @@
+/*
```

Comment:
Can we rename the class name? IMO, we do not use imperative forms for naming classes. Better to refer to some documentations regarding naming conventions.